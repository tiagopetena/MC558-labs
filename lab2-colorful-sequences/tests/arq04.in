50 100
48 49 0
27 47 1
47 26 1
16 7 0
45 21 0
34 8 1
46 47 0
41 1 0
31 9 0
48 46 1
35 49 0
49 22 1
26 48 0
21 2 0
37 36 1
32 14 1
2 13 1
20 26 1
35 9 0
5 43 1
25 27 1
27 40 0
48 29 1
43 33 0
46 7 1
25 31 1
19 30 1
1 46 0
28 17 1
8 5 0
43 41 1
26 21 0
14 42 0
9 1 1
21 39 1
7 2 1
5 20 0
32 36 1
3 34 0
18 12 1
47 18 0
21 11 1
4 21 1
36 20 0
22 25 0
40 34 0
9 37 1
40 23 1
39 2 0
8 20 0
39 35 0
33 8 1
49 3 1
12 40 1
6 33 1
6 22 0
38 35 1
45 16 1
4 40 0
22 26 1
2 18 1
17 43 0
24 19 0
8 6 0
15 44 0
10 15 1
0 46 0
37 48 0
30 28 0
33 4 1
23 32 0
33 23 0
34 24 1
1 14 1
20 19 1
9 27 1
23 8 1
20 6 1
17 16 1
36 45 0
26 12 0
18 4 0
35 45 1
27 32 0
8 38 0
46 39 1
16 25 0
7 8 1
14 25 0
42 0 1
7 9 0
29 27 0
40 35 1
44 5 1
11 7 0
19 37 0
12 10 0
25 48 1
2 17 0
13 33 0
