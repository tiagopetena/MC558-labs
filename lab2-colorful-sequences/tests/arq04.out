0 42 14 32 23 40 27 47 46 48 49 22 25 27 32 36 20 26 48 29 27 9 31 25 16 17 2 18 4 33 23 8 20 6 8 7 16 45 21 39 2 13 33 8 5 43 33 6 22 26 21 11 7 46 1 9 35 38 8 34 3 49 35 45 36 37 48 25 14 1 41 43 17 28 30 19 24 34 40 12 26 47 18 12 10 15 44 5 20 19 37 9 7 2 21 4 40 35 39 46 0
